app/grandchallenge/retina_api/mixins.py

```diff
@@ -45,13 +45,17 @@ class RetinaOwnerAPIPermission(permissions.BasePermission):
     """
     Permission class for annotation APIViews in retina app.
     Checks if user is in retina admins group or is owner of this object.
+    Requires that user_id is in view.kwargs to check for ownership.
     """
 
     def has_permission(self, request, view):
         user_id = view.kwargs.get("user_id")
         user = get_user_model().objects.get(id=user_id)
-        is_owner = request.user == user
-        return is_in_retina_admins_group(request.user) or is_owner
+        is_retina_user_and_owner = request.user == user and is_in_retina_group(
+            request.user
+        )
+        is_admin = is_in_retina_admins_group(request.user)
+        return is_admin or is_retina_user_and_owner
```

Comment:
Just a small style thing, line 54 was hard to grok, could you expand this out into 

```
if request.user == user and is_in_retina_group(request.user):
  return True
elif is_in_retina_admins_group(request.user):
  return True
else:
  return False
```

I know it's more verbose but I think it's much easier for a future dev to see whats going on immediately.

Reply:
Done in c2f4ef79527de7ca30c3884cb11cf487f8a8269b